docs/components/date-picker.md

```diff
@@ -3,51 +3,119 @@ layout: layouts/component.njk
 title: Date picker
 ---
 
-<span class="govuk-caption-xl">This component has recently been contributed to the MoJ Design System and is being developed.</span>
+<span class="govuk-caption-xl">The date picker component enables users to select a date from a calendar.  </span>
 
-## Status of development
+{% example "/examples/date-picker", 590 %}
 
-The below criteria all need to be met for a component to be considered as fully developed for use within the MoJ Design System.
 
-This page will be updated as the component is developed.
+## Overview
+
+When users first open the date picker's calendar it'll show today's date. Users do not have to use the calendar view to select a date - they can also enter one directly into the text field.
+
+### When to use
+
+Users might want to use the calendar view:
+
+- for a relative date or one they need to look up, for example last Thursday or next Wednesday
+- to enter today's date more quickly
+- for available dates only, for example for prison visits
+
+### When not to use
+
+Do not use the date picker:
+
+- for a memorable date, for example a user's date of birth
+- for a date that users know or can easily look up, for example an appointment on a letter
+- when only a rough date is needed, for example a month and year
+
+Use the [GOV.UK Design System's date input component](https://design-system.service.gov.uk/components/date-input/) instead.
+
+### Things to consider
+
+Date pickers are fully navigable using a keyboard, but can be slow for keyboard-only and screen reader users.
+
+### Similar or linked components
+
+There's also the ['Ask users for dates' pattern in the GOV.UK Design System](https://design-system.service.gov.uk/patterns/dates/).
+
+
+## How to use
+
+### Hint text
+
+The date picker hint text is set to 17/5/2024. This can be changed to a more helpful date, for example the start of a scheme. Add a full stop at the end.
+
+### Excluding dates
+
+<!-- You can set allowed date ranges if you need a user to pick a date within a date range. Individual dates and date ranges can also be disabled in the calendar view.
+
+Users may type unavailable or disabled dates in the input field, so error messages will be necessary.
+
+{% example "/examples/date-picker-min-max", 590 %} -->
```

Comment:
Yep, its functionality we added, but are not documenting publicly right now (tbh I'm not sure of the reason, it happened while I was away) I think the intention is that it will be introduced in the future, hence why it is commented rather than deleted.